### PR TITLE
Instead of doing a rebase

### PR DESCRIPTION
**Pull request by vodriemf**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/fodupload/controllers/StaticScanController.java [hunk 1]
```diff
@@ -94,11 +90,6 @@ public boolean startStaticScan(final JobModel uploadRequest, final String notes)
                     .addQueryParameter("technologyStack", token.getTechnologyType())
                     .addQueryParameter("entitlementId", Integer.toString(assessmentType.getEntitlementId()))
                     .addQueryParameter("entitlementFrequencyType", Integer.toString(assessmentType.getFrequencyTypeId()))
-                    .addQueryParameter("isBundledAssessment", Boolean.toString(assessmentType.isBundledAssessment()))
-                    .addQueryParameter("doSonatypeScan", Boolean.toString(includeOpenSourceScan))
```
Comment (james-mf) [inline src/main/java/org/jenkinsci/plugins/fodupload/controllers/StaticScanController.java]: Was this removed for 314001?  It doesn't seem related.

Comment (vodriemf) [reply src/main/java/org/jenkinsci/plugins/fodupload/controllers/StaticScanController.java]: It was removed in another ticket. I had 2 or 3 PRs open at the same time.

Comment (james-mf) [reply src/main/java/org/jenkinsci/plugins/fodupload/controllers/StaticScanController.java]: Didn't realize the merge is going from master to a branch.  This is fine.  Advanced options were removed elsewhere.
